include/libdnf5/conf/config_main.hpp

```diff
@@ -44,6 +44,8 @@ class LIBDNF_API ConfigMain : public Config {
 
     OptionNumber<std::int32_t> & get_debuglevel_option();
     const OptionNumber<std::int32_t> & get_debuglevel_option() const;
+    OptionNumber<std::int32_t> & get_errorlevel_option();
```

Comment:
Could you mark these functions as deprecated? I don't know if we have a dedicated document for the deprecations or a macro, but marking the prototypes with `__attribute((deprecated("errorlevel option does nothing and is scheduled for removal")))` would be great. This makes people using these methods to see a compile-time warning.

Reply:
That sounds good, I have also added a `@deprecated` since this is already used in the file.
Thank you.